src/components/organisms/nav_kalastatic/_nav_kalastatic.scss

```diff
@@ -99,7 +99,7 @@ $nav-bar-padding-shrunk: 1.5em;
   @media screen and (min-width: $screen-sm) {
     transition-property: padding-top;
     transition-duration: .25s;
-    @extend %easeOutOver;
+    /*@extend %easeOutQuad;*/
```

Comment:
Media queries don't work with extends. Before it would just not do anything. Looks like newer version errors out.